examples/patterns/navigation/subnav.html

```diff
@@ -0,0 +1,145 @@
+      </a>
+    </div>
+    <a href="#navigation" class="p-navigation__toggle--open" title="menu">Menu</a>
+    <a href="#navigation-closed" class="p-navigation__toggle--close" title="close menu">Close menu</a>
+  </div>
+  <nav class="p-navigation__nav">
+    <form class="p-search-box" action="/search">
+      <input type="search" class="p-search-box__input" name="q" placeholder="Search" required="">
+      <button type="reset" class="p-search-box__reset" alt="reset"><i class="p-icon--close"></i></button>
+      <button type="submit" class="p-search-box__button" alt="search"><i class="p-icon--search"></i></button>
+    </form>
+    <span class="u-off-screen">
+      <a href="#main-content">Jump to main content</a>
+    </span>
+    <ul class="p-navigation__links" role="menu">
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-1">
+        <a href="#link-1-menu" class="p-subnav__toggle">Link 1</i>
+        </a>
+        <ul class="p-subnav__items" id="link-1-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-2">
+        <a href="#link-2-menu" class="p-subnav__toggle">Link 2</i>
+        </a>
+        <ul class="p-subnav__items" id="link-2-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 4</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 5</a>
+          </li>
+        </ul>
+      </li>
+      <li class="p-navigation__link p-subnav" role="menuitem" id="link-3">
+        <a href="#link-3-menu" class="p-subnav__toggle">Link 3</a>
+        <ul class="p-subnav__items" id="link-3-menu" aria-hidden="true" style="display: none;">
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 1</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 2</a>
+          </li>
+          <li>
+            <a href="#" class="p-subnav__item">Sub item 3</a>
+          </li>
+        </ul>
+      </li>
+    </ul>
+  </nav>
+</header>
+
+<script>
+var dropdowns = document.querySelectorAll('.p-subnav__items');
+var dropdownToggles = document.querySelectorAll('.p-subnav');
+var dropdownToggleLinks = document.querySelectorAll('.p-subnav__toggle');
```

Comment:
These variables do not match what the query is returning anymore.

Reply:
Updated